dc-model/pom.xml

```
@@ -18,7 +18,8 @@
     <version.assertj-core>3.11.1</version.assertj-core>
     <version.commons-io>2.6</version.commons-io>
     <version.guava>27.0.1-jre</version.guava>
-    <version.junit-jupiter-api>5.3.1</version.junit-jupiter-api>
+    <version.hamcrest>1.3</version.hamcrest>
+    <version.junit-jupiter-api>5.3.2</version.junit-jupiter-api>
```

vereinheitlichen: ohne "-api"
```suggestion
    <version.junit-jupiter>5.3.2</version.junit-jupiter>
```